Views/MenuWidget.cshtml

```diff
@@ -0,0 +1,83 @@
+@using Lombiq.HelpfulExtensions.Extensions.Widgets
+@using Lombiq.HelpfulExtensions.Extensions.Widgets.ViewModels
+
+<style asp-name="font-awesome"></style>
+
+@{
+    var viewModel = new MenuWidgetViewModel(Model);
+
+    const string blockName = WidgetTypes.MenuWidget;
+    const string contentElementName = blockName + "__content";
+    const string togglerElementName = blockName + "__toggler";
+    const string dropdownElementName = blockName + "__dropdown";
+    const string dropdownItemElementName = blockName + "__dropdownItem";
+
+    var contentIdName = FormattableString.Invariant($"{contentElementName}_{Guid.NewGuid()}");
+    var dropdownIdName = FormattableString.Invariant($"{dropdownElementName}_{Guid.NewGuid()}");
+}
+
+@if (!viewModel.NoWrapper)
+{
+    @:<nav class="navbar navbar-expand-md">

```

Comment:
@DAud-IcI Why do you need the `@:` here? The markup itself should interrupt the code block, no?

Reply:
You get an error that "Tag nav is not closed", becuase it thinks the HTML isn't well nested.

Reply:
😲